From 3461aeba7a3e6b28549018a751524f51fe13cde3 Mon Sep 17 00:00:00 2001
From: RDunkley <tester@localhost.localdomain>
Date: Sat, 28 Dec 2019 13:40:47 -0700
Subject: [PATCH] Updated to support TE0820-03-03eg board.

---
 arch/arm/dts/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 445ab1526b..6a8d2618f1 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -166,6 +166,7 @@ dtb-$(CONFIG_ARCH_ZYNQ) += \
 	zynq-zybo-z7.dtb
 dtb-$(CONFIG_ARCH_ZYNQMP) += \
 	avnet-ultra96-rev1.dtb			\
+	te0820-03-03eg.dtb			\
 	zynqmp-mini.dtb				\
 	zynqmp-mini-emmc0.dtb			\
 	zynqmp-mini-emmc1.dtb			\
